### PR TITLE
done with kth and tests passing

### PR DESCRIPTION
**Pull request by carlitos-1988**

### CODE REVIEW

**Comment by carlitos-1988** [comment]
- [X] Top-level README “Table of Contents” is updated
 - [X] README for this challenge is complete
       - [X] Summary, Description, Approach & Efficiency, Solution
       - [ ] Picture of whiteboard
       - [X] Link to code
 - [X] Feature tasks for this challenge are completed
 - [X] Unit tests written and passing
       - [X] “Happy Path” - Expected outcome
       - [ X] Expected failure
       - [X ] Edge Case (if applicable/obvious)

File: linkedList/app/src/main/java/linkedlist/LinkedList.java [hunk 1]
```diff
@@ -82,6 +88,48 @@ public void insertAfter(int searchValue, int newValue){
...
+
+        //create curr node for the tail
+        Node curr = this.tail;
+        //need int location counter
+        int locationCounter = 0;
+
+        // if it is there start array from tail and iterate backwards
+        while (curr != null){
+//             COMMENTED OUT COULD NOT PASS TEST
+//            if(locationCounter > k){
```
Comment (lucy-gelderloos) [inline linkedList/app/src/main/java/linkedlist/LinkedList.java:106]: This `if` statement will never be true inside this `while` loop. When `curr` reaches the head of the list, `locationCounter` will be the same length as the list. If `k` is longer than the length of the list, at this point, `locationCounter > k` will evaluate to `false`. On the next iteration, `curr` will be null, the `while` loop will break, and it will throw the exception on line 126.
